index.html

```diff
@@ -15,12 +15,18 @@
         <section class="header">
 
             <header class="toolbar ">
-                <a href="#" > <img class="menu-bar" src="images/Icon - Menu.png" alt="menu bar"></a>
-                <ul>
-                    <li><a href="#"> Protofolio</li></a>
-                    <li><a href="#"> About</li></a>
-                    <li><a href="#"> Contact</li></a>
+            
+              <a href="#">
+                <span id="menuIcon" class="menu-bar">&#9776;</span>
+                <span id="closeIcon" class="close-icon">&times;</span>
```

Comment:
- [ ] Kindly change the color of your close button on the menu to that provided on the template 👍 . This is to show your skills in recreating a given design 👍 

###  Figma:
![image](https://github.com/Nessrine88/Portfolio.github.io/assets/50990925/72f43ab9-13b1-434e-a8a6-2d479871b163)

### Your work:
![image](https://github.com/Nessrine88/Portfolio.github.io/assets/50990925/34a23874-c0f3-4037-99d6-25554c6d8119)